main.storage.tf

```diff
@@ -0,0 +1,41 @@
+resource "azurerm_private_dns_zone" "storage_dns_zones" {
+  for_each = local.storage_account_endpoints
+
+  name                = "privatelink.${each.value}.core.windows.net"
+  resource_group_name = var.resource_group_name
+}
+
+module "avm_res_storage_storageaccount" {
+  source = "Azure/avm-res-storage-storageaccount/azurerm"
+  enable_telemetry              = var.enable_telemetry
+  // regex to remove invalid characters from the name
+  name                          = local.storage_account_name
+  resource_group_name           = var.resource_group_name
+  location                      = var.location
+  account_replication_type      = "GRS"
+  account_tier                  = "Standard"
+  account_kind                  = "StorageV2"
```

Comment:
DO you want those to be variables that cna be controled or do you think that if someone wnats to control this they should pass in their own storage account ?